src/main/java/org/commcare/formplayer/session/MenuSession.java

```diff
@@ -192,21 +192,33 @@ private String getReferenceToLatest(String appId) {
     }
 
     public boolean handleInput(String input) throws CommCareSessionException {
-        return handleInput(input, true);
+        return handleInput(input, true, false);
     }
 
     /**
      * @param input the user step input
      * @return Whether or not we were able to evaluate to a new screen.
      */
-    public boolean handleInput(String input, boolean needsDetail) throws CommCareSessionException {
+    public boolean handleInput(String input, boolean needsDetail, boolean confirmed) throws CommCareSessionException {
```

Comment:
Can you put the 'confirmed' param in the method doc string above along with its purpose and behavior?